src/main/java/de/tum/in/www1/artemis/service/user/UserCreationService.java

```diff
@@ -121,8 +123,12 @@ public User createInternalUser(String login, @Nullable String password, @Nullabl
         // needs to be mutable --> new HashSet<>(Set.of(...))
         final var authorities = new HashSet<>(Set.of(authority));
         newUser.setAuthorities(authorities);
-        Set<Organization> matchingOrganizations = organizationRepository.getAllMatchingOrganizationsByUserEmail(email);
-        newUser.setOrganizations(matchingOrganizations);
+        try {
+            Set<Organization> matchingOrganizations = organizationRepository.getAllMatchingOrganizationsByUserEmail(email);
+            newUser.setOrganizations(matchingOrganizations);
+        } catch (InvalidDataAccessApiUsageException | PatternSyntaxException pse) {
+            log.warn("Could not retrieve matching organizations from pattern: " + pse.getMessage());
```

Comment:
```suggestion
            log.warn("Could not retrieve matching organizations from pattern: {}", pse.getMessage());
```
In other places the formatting functionality of the logger is used when possible instead of string concatenation.